app/appshell/app.js

```diff
@@ -210,8 +211,8 @@ app.getRemoteDebuggingPort = function () {
 };
 
 app.getUserDocumentsDirectory = function () {
-    // TODO: implement
-    throw new Error("app.getUserDocumentsDirectory not implemented");
+    var _path = path.resolve(electronApp.getPath("home"), "Documents");
```

Comment:
I would suggest doing a Linux-specific call that removes the `"Documents"` append. Not everyone has their home dirs set up like that on Linux.

go with:
```javascript
var _path = path.resolve(electronApp.getPath("home"), "Documents");
```